dashboard/src/components/cfp-public/SubmissionsListView.vue

```
@@ -57,10 +57,29 @@ watch(
-  },
+    const search = searchTitle.value.trim().toLowerCase()
+    let filtered = submissions.originalData
+
+    // Apply basic field filters (status, session_type, etc.)
+    if (filters.value) {
+      filtered = filterSubmissions(filtered, filters.value)
+    }
+
+    // Apply search on talk title or speaker name
+    if (search) {
+      filtered = filtered.filter(submission => {
```

ESLint

[error] Replace `submission` with `(submission)`

(prettier/prettier)